### DISCUSSION TITLE
Decouple Results From Static Templates

### DISCUSSION THREAD

**Discussion by bcolflesh** [digitalfortress-tech/typeahead-standalone#89] [Q&A]
When I saw this had templates, I was thinking it meant you could specify something in the DOM and use that as a result container, but then I read the docs and looked at the playground examples.

I want to use the results to update my flexbox below the search input with video on demand results that include an image and the title being returned from my remote JSON file. Is there an example that shows something similar to this that someone can drop a link to?

**Comment by niketpathak** [comment]
@bcolflesh Currently, this is not a possibility since the search container is tied to the search input and injected in such a way that it floats below the input. However, this can be considered as a option for a future release. Kindly open a [feature request](https://github.com/digitalfortress-tech/typeahead-standalone/issues) if necessary.
